h/templates/accounts/notifications.html.jinja2

```diff
@@ -1,13 +1,23 @@
 {% extends "h:templates/layouts/account.html.jinja2" %}
 
+{% if not feature('activity_pages') %}
+{% set uses_legacy_styles = True %}
+{% endif %}
+
 {% set page_route = 'account_notifications' %}
 {% set page_title = 'Notifications' %}
 
 {% block page_content %}
-  <div class="form-vertical">
-    <h2 class="form-heading">
-      <span>{% trans %}Update your notification settings{% endtrans %}</span>
-    </h2>
-    {{ form }}
-  </div>
+  {% if feature('activity_pages') %}
+    <div class="form content">
```

Comment:
`form` and `content` are two separate blocks here. In older templates, the `content` class was used to denote the main content of the page. We don't actually style this class at all in the new CSS at the moment, so I would suggest that we just leave it out for the moment.